recipes/outcome/all/submoduledata.yml

```diff
@@ -0,0 +1,105 @@
+submodules:
+  "2.2.4":
+    status-code:
+      url: https://github.com/ned14/status-code/archive/38e1e862386cb38d577664fd681ef829b0e03fba.zip
+      sha256:
+      destination: ["include", "outcome", "experimental", "status-code"]
+    quickcpplib:
+      url: "https://github.com/ned14/quickcpplib/archive/57f8c679a939728c171c8780e484e94711b3a7e3.zip"
+      sha256: "ef13512b39570a14baf0f514bd8f49e61a23bc7804292c7b6eab794ab528a80a"
+      destination: ["quickcpplib"]
+    Outcome:
+      url: "https://github.com/akrzemi1/Optional/archive/2b43315458a99fc5de1da6e7bc0ddd364b26d643.zip"
+      sha256: "9e9a1a80c019012c0d1359d144447bd4bc40e636f8c24c9161e588ed76b3579f"
+      destination: ["quickcpplib", "include", "quickcpplib", "optional"]
+  "2.2.3":
+    status-code:
+      url: https://github.com/ned14/status-code/archive/60ec9e46916e5170051293e7925729ba88951934.zip
```

Comment:
These are valid recipes that should be added to ConanCenter https://github.com/conan-io/conan-center-index/pull/11459 for example